sdks/python/apache_beam/testing/analyzers/tests_config.yaml

```diff
@@ -22,7 +22,8 @@ pytorch_image_classification_benchmarks-resnet152-mean_inference_batch_latency_m
   test_description:
     Pytorch image classification on 50k images of size 224 x 224 with resnet 152.
     Test link - https://github.com/apache/beam/blob/42d0a6e3564d8b9c5d912428a6de18fb22a13ac1/.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy#L63
-    Test dashboard - http://metrics.beam.apache.org/d/ZpS8Uf44z/python-ml-runinference-benchmarks?orgId=1&viewPanel=2
+    Test dashboard - http://35.193.202.176/ZpS8Uf44z/python-ml-runinference-benchmarks?orgId=1&viewPanel=2
```

Comment:
Are these changes to an IP instead of the metrics.beam.apache.org URL intended?

Reply:
oh no. I will revert this. Thanks